code/src/App.js

```diff
@@ -1,9 +1,74 @@
-import React from 'react'
+/*Outer Dependencies */
+import React, { useState, useEffect } from "react";
+
+/* Local Dependencies */
+import MessageForm from "./components/MessageForm";
+import MessageList from "./components/MessageList";
+
+import { API_URL, LIKES_URL } from "./reusable/urls";
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([]);
+  const [messageNew, setMessageNew] = useState("");
+
+  useEffect(() => {
+    fetchMessageList();
+  }, []);
+
+  const fetchMessageList = () => {
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((recievedMessages) => setMessageList(recievedMessages))
+      .catch((err) => console.error(err));
```

Comment:
console outputs can't be present in the production code.

Reply:
Ah! You are right, fixed those just now. :)